eng/versioning/external_dependencies.txt

```diff
@@ -158,7 +158,7 @@ org.springframework:spring-jdbc;5.3.27
 org.springframework:spring-jms;5.3.27
 org.springframework:spring-messaging;5.3.27
 org.springframework:spring-tx;5.3.27
-org.springframework:spring-web;5.3.27
+org.springframework:spring-web;6.0.0
```

Comment:
6.x is compatible with Spring Boot 3, however, the 5.x is compatible with Spring Boot 2. This isn't going to work.